### PR TITLE
[14.0][MIG] product_quantity_update_force_inventory

### PR DESCRIPTION
**Pull request by mariadforgeflow**
Standard Migration to v14.

### CODE REVIEW

**Review by JoanSForgeFlow — APPROVED** [review]
Functional review in my local. LGTM :+1:

File: product_quantity_update_force_inventory/models/product.py [hunk 1]
```diff
@@ -10,7 +10,7 @@ def action_update_quantity_on_inventory_adjustment(self):
         """Create an Inventory Adjustment instead of edit directly on quants"""
         self.ensure_one()
         view_form_id = self.env.ref("stock.view_inventory_form").id
-        action = self.env.ref("stock.action_inventory_form").read()[0]
+        action = self.env.ref("stock.action_inventory_form").sudo().read()[0]
```
Comment (HviorForgeFlow) [inline product_quantity_update_force_inventory/models/product.py:13]: Why sudo() has been added? Just curious :eyes:

Comment (mariadforgeflow) [reply product_quantity_update_force_inventory/models/product.py:13]: I got an Action Window permission error and I found [this was a way to fix it.](https://github.com/antonyht27/stock-logistics-workflow/commit/4d2013b5515e0e3e9503b9c85d27f16a52696a70) I'm not sure if it's the best way, suggestions are welcomed:)

**Review by HviorForgeFlow — APPROVED** [review]
OK, I see the error in when running the test, seems legit.

code + functional review :+1: LGTM

**Pull request merged**